src/main/kotlin/com/carneirofatima/vehicleterminal/service/DriverService.kt

```diff
@@ -0,0 +1,38 @@
+package com.carneirofatima.vehicleterminal.service
+
+import com.carneirofatima.vehicleterminal.dao.DriverDao
+import com.carneirofatima.vehicleterminal.entity.Driver
+import com.carneirofatima.vehicleterminal.to.DriverTO
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class DriverService(
+    private val dao: DriverDao
```

Comment:
this attribute name may be better. `driveDao` or `driveRepository` is more explicit